diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/caboodle-0.5/Makefile src/caboodle-0.5/Makefile
--- origsrc/caboodle-0.5/Makefile	2006-11-04 14:49:57.000000000 -0600
+++ src/caboodle-0.5/Makefile	2007-06-26 17:11:54.176768800 -0500
@@ -11,17 +11,16 @@
 
 build:
 	cd src && $(MAKE)	
-
-install: build
 	$(MKDIR) -p build
+	$(SED) "s|PREFIX|$(PREFIX)|" caboodle.desktop > build/caboodle.desktop
+
+install: build/caboodle.desktop
 	$(MKDIR) -m 755 -p $(PREFIX)/bin
 	$(MKDIR) -m 755 -p $(PREFIX)/share/caboodle
 	$(MKDIR) -m 755 -p $(DESKTOPLOCATION)
-	$(SED) "s|PREFIX|$(PREFIX)|" caboodle.desktop > build/caboodle.desktop
 	$(INSTALL) -m 644 share/caboodle/caboodle.glade share/caboodle/caboodle.svg $(PREFIX)/share/caboodle
 	$(INSTALL) -m 755 src/caboodle $(PREFIX)/bin
 	$(INSTALL) -m 644 build/caboodle.desktop $(DESKTOPLOCATION)
-	update-desktop-database $(DESKTOPLOCATION)
 	
 clean:
 	$(RM) -rf build
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/caboodle-0.5/src/commondefs.ml src/caboodle-0.5/src/commondefs.ml
--- origsrc/caboodle-0.5/src/commondefs.ml	2006-11-04 14:49:57.000000000 -0600
+++ src/caboodle-0.5/src/commondefs.ml	2007-06-26 17:14:13.927663200 -0500
@@ -18,7 +18,7 @@
 Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 *)
 
-let version = "0.4";;
+let version = "0.5";;
 
 type vertex = { 
     id : int;
