--- origsrc/caboodle-0.5/Makefile	2006-11-04 14:49:57.000000000 -0600
+++ src/caboodle-0.5/Makefile	2007-06-26 17:11:54.176768800 -0500
@@ -11,17 +11,16 @@
 
 build:
 	cd src && $(MAKE)	
-
-install: build
 	$(MKDIR) -p build
+	$(SED) "s|PREFIX|$(PREFIX)|" caboodle.desktop > build/caboodle.desktop
+
+install: build/caboodle.desktop
 	$(MKDIR) -m 755 -p $(PREFIX)/bin
 	$(MKDIR) -m 755 -p $(PREFIX)/share/caboodle
 	$(MKDIR) -m 755 -p $(DESKTOPLOCATION)
-	$(SED) "s|PREFIX|$(PREFIX)|" caboodle.desktop > build/caboodle.desktop
 	$(INSTALL) -m 644 share/caboodle/caboodle.glade share/caboodle/caboodle.svg $(PREFIX)/share/caboodle
 	$(INSTALL) -m 755 src/caboodle $(PREFIX)/bin
 	$(INSTALL) -m 644 build/caboodle.desktop $(DESKTOPLOCATION)
-	update-desktop-database $(DESKTOPLOCATION)
 	
 clean:
 	$(RM) -rf build
--- origsrc/caboodle-0.5/src/commondefs.ml	2006-11-04 14:49:57.000000000 -0600
+++ src/caboodle-0.5/src/commondefs.ml	2007-06-26 17:14:13.927663200 -0500
@@ -18,7 +18,7 @@
 Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 *)
 
-let version = "0.4";;
+let version = "0.5";;
 
 type vertex = { 
     id : int;
